### DISCUSSION TITLE
i want to make email address for each users in my website

### DISCUSSION THREAD

**Discussion by vahidalvandi** [docker-mailserver/docker-mailserver#3074] [Ideas]
How can I request to create an email account through the website? And when an email is sent to that account, it will be sent to the user on the website

**Comment by polarathene** [comment]
You need to have `postfix-accounts.cf` updated (_eg: by calling `setup email add ...`_), or having luck with the LDAP setup (_you should be familiar with LDAP for this, no maintainers have LDAP experience_). If that's not acceptable, you may want to look for alternative solutions than DMS.

> And when an email is sent to that account, it will be sent to the user on the website

This is not very clear what you're asking for. It does not sound within the scope of this project.
